File: VDS_Frontend/src/app/prueba/Models/prueba.dto.ts
export interface Prueba {
    id?: number;
    tipo: TipoPrueba;
    descripcion?: string;
    fecha: string;
    resultado?: string;
    consultaId: number;
    mascotaId: number;
  }
  
  export enum TipoPrueba {
    IMAGEN = 'IMAGEN',
    ANALÍTICA = 'ANALÍTICA'
  }

File: VDS_Backend/src/main/java/com/uoc/tfm/vds_backend/prueba/dto/PruebaDTO.java
package com.uoc.tfm.vds_backend.prueba.dto;

import java.time.LocalDateTime;
import java.util.List;

import com.uoc.tfm.vds_backend.prueba.model.TipoPrueba;

import lombok.Data;

@Data
public class PruebaDTO {
    private Long id;
    private TipoPrueba tipo;
    private String descripcion;
    private LocalDateTime fecha;
    private Long mascotaId;
    private Long consultaId;
    private List<Long> documentoPruebaIds;
}
